### ISSUE TITLE
Issues with the Scheduler

### ISSUE THREAD

**Issue by benhayes36** [promac2k/ClanWarsSchedule#1]
ProMac

Been having issues getting this to work.  From what I can tell there is issues due to the recent screen resolution being changed (at least that is what I am seeing).  The Scheduler loads the correct emulator and Instance but at that point, it fails to load Clash of Clans (due to the screen resolution).  I've tried several different ways of getting this to run without any success.  Can you have a look at this please.  Sent a message on the forums for you with log file attached.

**Comment by benhayes36** [comment]
[Log.zip](https://github.com/promac2k/ClanWarsScheduler/files/3011115/Log.zip)

**Comment by promac2k** [comment]
Yes we changed from 860*732 to 860*644 and i didn't make the correct changes! next weekend i will do it, thks for your report. 

> 2019-03-26 21:21:17 - yes yes 
> 2019-03-26 21:22:35 - Window Name: nox
> 2019-03-26 21:22:35 - Emulator ScreenSizes ERROR!!
> 2019-03-26 21:22:36 - Selected 50 War Participant(s)

**Comment by promac2k** [comment]
Zip deleted

**Comment by promac2k** [comment]
https://github.com/promac2k/ClanWarsScheduler/tree/hotfix/ClanWarsScheduler_v1.1.1

**Comment by benhayes36** [comment]
ProMac

Still having issues - even with the latest version:

2019-03-28 22:27:13 - Welcome back benha, Good Evening
2019-03-28 22:27:13 - OS Type: X64
2019-03-28 22:27:13 - Version: WIN_10
2019-03-28 22:27:13 - Build: 17763
2019-03-28 22:27:13 - AMD Ryzen 5 1600 Six-Core Processor            
2019-03-28 22:27:13 - MaxClockSpeed: 3200
2019-03-28 22:27:13 - Virtualization Support : True
2019-03-28 22:27:13 - Virtualization Enable : True
2019-03-28 22:27:13 - MEmu is not installed on your system
2019-03-28 22:27:13 - Selected 50 War Participant(s)
2019-03-28 22:27:18 - Selected Nox
2019-03-28 22:27:30 - Please change the Hours Or/And Minutes first!
2019-03-28 22:27:37 - Selected Emulator is Nox
2019-03-28 22:27:37 -  »»»»» Opening Emulator 'TESTS'
2019-03-28 22:27:37 - Selected 'nox' Instance is Opening
2019-03-28 22:27:37 - Wait a few seconds...
2019-03-28 22:27:42 - yes yes 
2019-03-28 22:27:42 - [1] Nox Child Window detected: 0x008D0676
2019-03-28 22:27:42 - Window Name: Johno (CoC)
2019-03-28 22:27:42 - Emulator Size: [W]: 864  [H]: 30
2019-03-28 22:27:42 - Emulator ScreenSizes ERROR!!

It loads the Nox Instance but doesn't do anything more than that.

**Comment by promac2k** [comment]
Nox version?

**Comment by benhayes36** [comment]
v6.2.1.1. I'm also using the latest beta release (Fenix Mod).

**Comment by benhayes36** [comment]
Promac wanted to check in to see if there has been any updates to the tool.
